pom.xml

```diff
@@ -160,6 +160,20 @@
             </modules>
         </profile>
 
+        <profile>
+            <id>2.0-no-example</id>
```

Comment:
Question, do we still need to have this `2.0` profile?

Reply:
@symphony-thibault or directly exclude the example module out of profile `2.0`, then we don't need to update the CircleCI workflow